package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uswds-docs",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Oh, good question. For CALC we only versioned on pushes from `develop` to `master`... So it's similar to your current workflow, I think, only that we do the version bump in PRs like #320, and we also update the [`CHANGELOG.md`](https://calc.gsa.gov/docs/changelog.html) in that PR too.  It's documented in our [release process](https://calc.gsa.gov/docs/release.html) which is a little more heavyweight than I'd like it to be, I guess, but it works.